docs/serial_console.md

```diff
@@ -87,13 +91,43 @@ below `*** Serial port base address = 0x3f8 ***` will change to
 legacy). Then go to Payload menu and type the changed serial port base address
 (`0x2f8`) to `SeaBIOS sercon-port base address`  field. Now save new config.
 
+### COM2 runtime configuration
+
+Since v4.8.0.7 and v4.0.22, sortbootorder has a new option to enable output
+redirection to COM2. After entering sortbootorder menu, one could notice
+additional COM2 redirection among other options:
+
+```
+  u USB boot - Currently Enabled
+  t Serial console - Currently Enabled
+  k Redirect console output to COM2 - Currently Disabled
+  o UART C - Currently Enabled
+  p UART D - Currently Enabled
```

Comment:
@miczyg1 I assume sortbootorder documentation was update accordingly?

Reply:
@pietrushnic we do not have sortbootorder documentation

Reply:
@miczyg1 please add that to sortbootorder documentation: https://github.com/pcengines/sortbootorder - we should always keep consistency

Reply:
You could say sortbootorder README. Done.